src/web/index.css

```
@@ -6764,33 +6764,58 @@ textarea {
 
 .route-detail-panel-presence.is-open .route-card-detail-panel {
-  animation: route-detail-panel-in 0.18s cubic-bezier(0.22, 1, 0.36, 1);
+  animation: route-detail-panel-in 0.26s cubic-bezier(0.22, 1, 0.36, 1) both;
+}
+
+.route-detail-panel-presence.is-closing .route-card-detail-panel {
+  animation: route-detail-panel-out 0.2s cubic-bezier(0.4, 0, 1, 1) both;
 }
 
 @keyframes route-detail-panel-in {
   from {
```

Stylelint (17.6.0)

[error] Expected "from" to be "0%" (keyframe-selector-notation)

(keyframe-selector-notation)